packages/discord.js/typings/index.d.ts

```diff
@@ -4063,7 +4063,14 @@ export interface CommandInteractionResolvedData<Cached extends CacheType = Cache
   attachments?: Collection<Snowflake, Attachment>;
 }
 
-export type AutocompleteOption = Pick<CommandInteractionOption, 'name' | 'type' | 'value' | 'focused'>;
+export type AutocompleteOption = Pick<CommandInteractionOption, 'name'> & {
+  type:
+    | ApplicationCommandOptionType.String
+    | ApplicationCommandOptionType.Integer
+    | ApplicationCommandOptionType.Number;
+  focused: boolean;
+  value: string | number;
+};
```

Comment:
```suggestion
export type AutocompleteOption = Required<Pick<CommandInteractionOption, 'name' | 'focused'>> & {
  type:
    | ApplicationCommandOptionType.String
    | ApplicationCommandOptionType.Integer
    | ApplicationCommandOptionType.Number;
  value: string | number;
};
```

Reply:
sure ig?

Reply:
`focused` is always a `true` here actually, so I don't think getting the optional property here would be accurate?

Reply:
Right, this is only used in `getFocused()`. The type could have a better name then

Reply:
oh yeah I thought it would be false for autocomplete options that aren't focused. In that case I'll revert this and make it true